index.html

```diff
@@ -52,114 +52,86 @@ <h1>Personal Profile</h1>
 				</div>
 				
 				<div class="sectionContent">
-					<p>I am a former registered nurse still licensed in both California and Texas. I am looking to make a career change into software development.</p>
+					<p>I am a former registered nurse looking to make a career change into software development.</p>

```

Comment:
I think this should be more direct.

"I'm a former registered nurse turned software developer."